dashboard/dashapi/dashapi.go

```diff
@@ -400,6 +401,11 @@ type BugReport struct {
 	BisectCause    *BisectResult
 	BisectFix      *BisectResult
 	Assets         []Asset
+	Subsystems     []*BugSubsystem
+}
+
+type BugSubsystem struct {
+	Name string
 }
```

Comment:
Why do we need 2 copies? Second one is in the dashboard/app/entities.go.

Reply:
I think it's better not to separate the structure that is a part of the API and the one that's stored in the DB. As we add more fields to the DB, we might not want to return them all via `dashapi.BugReport`.